### DISCUSSION TITLE
Generate only the win-x64 Magick native folder

### DISCUSSION THREAD

**Discussion by Yac-Mc** [dlemstra/Magick.NET#1739] [Help] [answered]
At the time of publishing it is generating the following folders:

- linux-musl-x64
- linux-x64
- osx-x64
- win-arm64
- win-x64
- win-x86

I just want to generate the native Magick win-x64 folder.

I tried with:
```
<PropertyGroup>
  <MagickCopyNativeWindows>true</MagickCopyNativeWindows>
  <MagickCopyNativeLinux>false</MagickCopyNativeLinux>
  <MagickCopyNativeLinuxMusl>false</MagickCopyNativeLinuxMusl>
  <MagickCopyNativeMacOS>false</MagickCopyNativeMacOS>
</PropertyGroup>
```
But it doesn't work :(

**Any ideas?**

**Comment by dlemstra** [comment, marked as answer]
What happens when you add `<RuntimeIdentifiers>win-x64</RuntimeIdentifiers>` in your `.csproj` file.

**Reply by Yac-Mc** [reply]
It works! Thank you

MagickCopyNativeWindows properties are not required
